packages/office-ui-fabric-react/src/components/Overlay/Overlay.styles.ts

```diff
@@ -0,0 +1,52 @@
+import { IOverlayStyleProps, IOverlayStyles } from './Overlay.types';
+import {
+  IStyle,
+  ITheme,
+  HighContrastSelector,
+} from '../../Styling';
+
+export const getStyles = (
+  props: IOverlayStyleProps
+): IOverlayStyles => {
+  const {
+    className,
+    theme,
+    isNone,
```

Comment:
Could this be renamed for clarity? Maybe reverse it and call it `isVisible`?

Reply:
Agreed, it's not used at the moment so that should be remedied.

Reply:
Can we remove it entirely then?

Reply:
I had the same question but forgot to follow up on it. @battletoilet - thoughts?